### ISSUE TITLE
Add repeatable alarm, down to every N minutes

### ISSUE THREAD

**Issue by ElSamhaa** [BlackyHawky/Clock#155]
I want to be able to schedule a repeatable alarm every 1 hour for example.

**Comment by BlackyHawky** [comment]
@ElSamhaa 
I may have misunderstood your request, but it's already possible in the _"Alarms"_ settings.

**Comment by ElSamhaa** [comment]
hi @BlackyHawky , I've updated the op

**Comment by BlackyHawky** [comment]
Indeed, currently, the choice of the number of minutes to repeat an alarm is not customizable; perhaps I could replace the `ListPreference` with an `EditTextPreference`?
The user could choose his own value in minutes... 🤔

**Comment by Ram-Z** [comment]
Feels more like a timer feature... Add a repeat checkbox to the timer. Maybe with a count.

**Comment by rimugu** [comment]
A vote for repeatable timers.

**Comment by DJCrashdummy** [comment]
IMHO this seems to rather be a - slightly less advanced - duplicate of #93.
...perhaps my suggestion in this comment (https://github.com/BlackyHawky/Clock/issues/93#issuecomment-2781364251) is of use for some of you.
